src/saga/adaptors/shell/shell_job.py

```diff
@@ -174,7 +174,17 @@ def _decode (data) :
                           suitable jobs, including the ones managed by another,
                           live job service instance.''',
     'env_variable'     : None
-}
+    },
+    {
+    'category'         : 'saga.adaptor.shell_job',
+    'name'             : 'base_workdir',
+    'type'             : str,
+    'default'          : "$HOME/.saga/adaptors/shell_job/",
+    'documentation'    : '''The adaptor stores job state information on the
+                          filesystem on the target resource.  This parameter
+                          specified what location should be used.''',
+    'env_variable'     : None
+    }
```

Comment:
Yeah, you can do without I gues, but then you have no type conversion, default value, documentation, env support... :)